css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: static;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 15rem;
```

Comment:
Once you fix the position for `.meme`, this can be done using `right` like I mentioned with a very small measurement.